lighthouse-core/audits/accessibility/bypass.js

```diff
@@ -23,7 +23,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Adding ways to bypass repetitive content lets keyboard users navigate the ' +
       'page more efficiently. ' +
-      '[Learn more](https://dequeuniversity.com/rules/axe/4.4/bypass).',
+      '[Learn more about bypass navigation mechanisms](https://dequeuniversity.com/rules/axe/4.4/bypass).',
```

Comment:
bypassing?

Reply:
or: Learn more about how to bypass ...

Reply:
The right terminology seems to be [`bypass blocks`](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks). So may be `Learn more about bypass blocks` or `Learn how to provide a bypass block`.